cli/package.json

```diff
@@ -20,10 +20,11 @@
     "react-native",
     "react"
   ],
+  "homepage": "https://microsoft.github.io/code-push",
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "code-push": "1.0.0-beta",
+    "code-push": "^1.1.0-beta",
```

Comment:
Will the current changes in the CLI work with SDK version 1.1.0-beta? If there are breaking changes, we should not use caret ranges, but direct dependency on `1.1.1-beta`.  `^1.1.0-beta` means `>=1.1.0-beta` and `< 2.0.0`